docker-compose.yml

```diff
@@ -166,7 +166,7 @@ services:
 
   preprints:
     image: quay.io/centerforopenscience/ember-preprints:develop
-    command: /bin/bash -c "npm install && ./node_modules/bower/bin/bower install --allow-root --config.intedractive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
+    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
```

Comment:
Our other Dockerfiles needed to install Yarn. Can you clarify why the OSF does not need this change?

Also: other PRs activating Yarn do not use some of these flags. If we find they are valuable, we might want to use the same consistent commands / set of flags across all of our repos.